### PR TITLE
UIIN-1103 Show two instance records as a split screen

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIIN-1103

## Approach

1. Instance details component to avoid copy paste on new screen
2. New route that displays 2 instances 

#### TODOS and Open Questions

move items/holding to separate routes to support them on split screen

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by NikitaSedyx** [comment]
old are tests are failed - after refactoring correct values are expected :)
e.g. previously instance title was expected in holding title, but it's not correct

File: src/ViewInstance.js [hunk 1]
```diff
@@ -523,236 +493,80 @@ class ViewInstance extends React.Component {
...
+            <Col sm={12}>{newHoldingsRecordButton}</Col>
+          </Row>
 
-            <InstanceSubjectView
-              id="acc08"
-              subjects={subjects}
-            />
+          {
+            (holdingsrecordid && !itemid)
+              ? (
```
Comment (VictorSoroka1) [inline src/ViewInstance.js:546]:
```suggestion
              && (
```

File: src/ViewInstance.js [hunk 2]
```diff
@@ -523,236 +493,80 @@ class ViewInstance extends React.Component {
...
-                        <this.cViewMarc
+      <>
+        <InstanceDetails
+          onClose={onClose}
+          actionMenu={this.createActionMenuGetter(instance)}
+          instance={instance}
+          referenceData={referenceTables}
+        >
+          {
+            (!holdingsrecordid && !itemid) ?
```
Comment (VictorSoroka1) [inline src/ViewInstance.js:505]:
```suggestion
            (!holdingsrecordid && !itemid) &&
```

File: src/ViewInstance.js (hunk 1 above)
Comment (NikitaSedyx) [reply src/ViewInstance.js:546]: will skip for now as these blocks will be moved from `ViewInstance` in next PRs

File: src/ViewInstance.js (hunk 2 above)
Comment (NikitaSedyx) [reply src/ViewInstance.js:505]: will skip for now as these blocks will be moved from `ViewInstance` in next PRs

File: src/Instance/InstanceMovement/useInstance.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+} from 'react';
+
+const useInstance = (id, mutator) => {
+  const [instance, setInstance] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    mutator.GET({
```
[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/Instance/InstanceMovement/useInstance.js:13]: no reason to do this as data is stored in state

File: src/routes/InstanceMovementRoute.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      idFrom={match.params.idFrom}
+      idTo={match.params.idTo}
+    />
+  );
+};
+
+InstanceMovementRoute.propTypes = {
+  match: PropTypes.object.isRequired,
+  isLoading: PropTypes.func,
+  getData: PropTypes.func,
```
[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/routes/InstanceMovementRoute.js:29]: yep

File: src/Instance/InstanceMovement/useInstance.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (VictorSoroka1) [reply src/Instance/InstanceMovement/useInstance.js:13]: Despite it is stored in state for one record, the data is also stored in redux-store. This what I meant.

[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/Instance/InstanceMovement/useInstance.js:13]: I think they will be overridden with new response

[the remark this answers is not in the document]
Comment (VictorSoroka1) [reply src/Instance/InstanceMovement/useInstance.js:13]: Sounds good, if this is thue.

[the remark this answers is not in the document]
Comment (VictorSoroka1) [reply src/Instance/InstanceMovement/useInstance.js:13]: So this is the screen after I opened the spit screen three times. One for the `idFrom` and another one `idTo` = six total.

Screen Shot 2020-07-02 at 21 29 12

**Pull request merged**